lib/util/mp4_generator.js

```diff
@@ -1230,6 +1300,10 @@ shaka.util.Mp4Generator.DINF_ = new Uint8Array([]);
  *    stream: !shaka.extern.Stream
  * }}
  *
+ * @property {number} id
+ *   A unique ID
+ * @property {string} type
+ *   Indicate the content type: 'video' or'audio'.
```

Comment:
And there should be a space before `'audio'`.

Reply:
Fixed!